sql/2020/22_HTTP_2/adoption_of_http_2_by_site_and_requests.sql

```diff
@@ -0,0 +1,20 @@
+#standardSQL
+# 22_HTTP_2 - Adoption of HTTP/2 by site and requests
+SELECT 
+  client,
+  firstHtml, 
+  JSON_EXTRACT_SCALAR(payload, "$._protocol") AS http_version, 
+  COUNT(*) AS num_requests,
+  ROUND(COUNT(0) * 100 / SUM(COUNT(0)) OVER (PARTITION BY client), 2) AS pct
```

Comment:
One change from last year is to use decimals for percents rather than `* 100`. This makes it easier to format with the % suffix in Sheets for use in data viz.
```suggestion
  ROUND(COUNT(0) / SUM(COUNT(0)) OVER (PARTITION BY client), 4) AS pct
```

Reply:
Sounds good. I made the change to all files that had similar constructs